packages/tpetra/core/src/Tpetra_Details_makeColMap_def.hpp

```diff
@@ -56,12 +56,272 @@
 #include "Tpetra_RowGraph.hpp"
 #include "Tpetra_Util.hpp"
 #include "Teuchos_Array.hpp"
+#include "Kokkos_Bitset.hpp"
 #include <set>
 #include <vector>
 
 namespace Tpetra {
 namespace Details {
 
+template <class LO, class GO, class NT>
+int
+makeColMapImpl (Teuchos::RCP<const Tpetra::Map<LO, GO, NT> >& colMap,
```

Comment:
@mhoemmen Yes to the first, no to the second - it's only an output. If colMap is non-null on input to ``makeColMap()`` then it will be clobbered with the new map.